ui/lib/apps/SlowQuery/translations/en.yaml

```diff
@@ -34,22 +34,45 @@ slow_query:
     db: Execution Database
     db_tooltip: The database used to execute the query
 
-    parse_time: Parse Time
-    parse_time_tooltip: Time consumed when parsing the query
-    compile_time: Compile Time
-    compile_time_tooltip: Time consumed when optimizing the query
-    wait_time: Coprocessor Wait Time
-    process_time: Coprocessor Process Time
-    backoff_time: Backoff Retry Time
-    backoff_time_tooltip: The waiting time before retry when a query encounters errors that require a retry
-    get_commit_ts_time: Get Commit Ts Time
-    local_latch_wait_time: Local Latch Wait Time
-    resolve_lock_time: Resolve Lock Time
-    prewrite_time: Prewrite Time
-    commit_time: Commit Time
-    commit_backoff_time: Commit Backoff Time
     query_time2: Query Time
-    query_time2_tooltip: The execution time of a query (due to the parallel execution, it may be significantly smaller than the above time)
+    query_time2_tooltip: The elapsed wall time when execution the query
+    parse_time: 　　Parse Time
+    parse_time_tooltip: Time consumed when parsing the query
+    compile_time: 　　Generate Plan Time
+    rewrite_time: 　　　　Rewrite Plan Time
+    preproc_subqueries_time: 　　　　　　Preprocess Sub-Query Time
+    preproc_subqueries_time_tooltip: Time consumed when pre-processing the subquery during the rewrite plan phase
+    optimize_time: 　　　　Optimize Plan Time
+    wait_ts: 　　Get Start Ts Time
+    wait_ts_tooltip: Time consumed of retrieving the transaction start timestamp when transaction begins
+    cop_time: 　　Coprocessor Executor Time
+    cop_time_tooltip: The elapsed wall time when TiDB Coprocessor executor waiting all Coprocessor requests to finish (note: when there are JOIN in SQL statement, multiple TiDB Coprocessor executors may be running in parallel, which may cause this time not being a wall time)
+    wait_time: 　　　　Coprocessor Wait Time
+    wait_time_tooltip: The total waiting time of Coprocessor request in TiKV
+    process_time: 　　　　Coprocessor Process Time
+    process_time_tooltip: The total time of Coprocessor request being executed in TiKV (note: TiKV executes requests in parallel so that this is not a wall time)
+    backoff_time: 　　Backoff Retry Time
+    backoff_time_tooltip: The total waiting time before retry when a query encounters errors and requires a retry (note: there may be multiple backoff running in parallel so that this may not be a wall time)
```

Comment:
```suggestion
    backoff_time_tooltip: 'The total waiting time before retry when a query encounters errors and requires a retry (note: there may be multiple backoff running in parallel so that this may not be a wall time)'
```